lib/t_web/views/view_helpers.ex

```diff
@@ -51,7 +50,7 @@ defmodule TWeb.ViewHelpers do
     end
   end
 
-  defp process_labels(%{"labels" => labels} = page, version) do
+  defp process_labels(%{"labels" => labels} = page, version, screen_width) do
     # we do not add url for versions prior to 6.2.0 because they will be rendered incorrectly
     labels =
       case Version.compare(version, "6.2.0") do
```

Comment:
needs another version check, maybe against v6.3.0?

Reply:
may do it, but not sure if I should since the "video" (unknown question) is "rendered" to nil in 6.2.0